python/nav/eventengine/topology.py

```diff
@@ -263,7 +263,7 @@ def get_source_address_for(dest):
     sock = socket.socket(family, socket.SOCK_DGRAM)
     try:
         sock.connect(sockaddr)
-    except socket.error as err:
+    except socket.error:
```

Comment:
Actually, there's a "hidden" use of the `err` variable in the subsequent log message: It feeds the entire `locals()` dict into the `_logger.warning()` call and uses the value of the `dest` and `err` local variables.

Not sure why it's done like this, but I would just change the entire log statement to explicitly feed in the used variables